druid/src/main/java/org/apache/calcite/adapter/druid/TimeExtractionDimensionSpec.java

```diff
@@ -50,33 +50,21 @@ public static TimeExtractionDimensionSpec makeFullTimeExtract(String outputName)
    * @return time field extraction DimensionSpec instance or null if granularity
    * is not supported
    */
-  public static TimeExtractionDimensionSpec makeExtract(
+  public static TimeExtractionDimensionSpec makeTimeExtract(
       Granularity granularity, String outputName) {
-    switch (granularity) {
-    case YEAR:
-      return new TimeExtractionDimensionSpec(
-          TimeExtractionFunction.createExtractFromGranularity(granularity), outputName);
-    case MONTH:
-      return new TimeExtractionDimensionSpec(
-          TimeExtractionFunction.createExtractFromGranularity(granularity), outputName);
-    case DAY:
-      return new TimeExtractionDimensionSpec(
-          TimeExtractionFunction.createExtractFromGranularity(granularity), outputName);
-    // TODO: Support other granularities
-    default:
-      return null;
-    }
+    return new TimeExtractionDimensionSpec(
```

Comment:
What if createExtractFromGranularity returns null? Is it valid for TimeExtractionDimensionSpec.extractionFunction to be null?